Rakefile

```diff
@@ -4,5 +4,20 @@ require 'middleman-gh-pages'
 require 'rspec/core/rake_task'
 RSpec::Core::RakeTask.new(:spec)
 
+desc "Compile javascript -> coffeescript"
+task :coffee do
+  destination = "source/javascripts/refills/coffeescript"
+  Dir["source/javascripts/refills/**/*.js"].each do |source|
+    filename = File.basename(source.gsub(%r{.js}, ".coffee"))
+
+    `js2coffee #{source} > #{destination}/#{filename}`
+  end
+
+  puts "Please make sure that all generated coffeescript files are correct"
+  puts "Then commit the changes"
+  puts
+  puts `git status`
```

Comment:
Does this assume that the only use case of this command is just before committing? Is that the only use case?